app/views/content/landing/campus-mailing-list.md

```diff
@@ -7,7 +7,6 @@ content:
 image: "static/images/content/hero-images/chemistry.jpg"
 colour: pink-yellow
 layout: "layouts/minimal"
-talk_to_us: false
```

Comment:
On the `/landing/campus-advisers` page, there is some breathing space between the body and the footer on mobile (albeit that it looks a bit stripy!)

However, on the `/landing/campus-mailing-list` page, there isn't the same kind of separation, which looks very odd?

@jenhadfield-dfe @spencerldixon 

![image](https://github.com/user-attachments/assets/6039a5f1-ed4e-4fbb-ba33-da6a08925b1c)

![image](https://github.com/user-attachments/assets/71663b42-29e1-4a69-8142-2f105ef1d0fe)

Reply:
@gemmadallmandfe I've added a `margin-top` to the footer to keep a bit of space between the main content and the footer, let me know what you think

Reply:
That looks better to me! @jenhadfield-dfe are you happy with this?

Reply:
yes looks much better!